cmake/DownloadMKLML.cmake

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This script will download MKLML
+
+message(STATUS "Downloading MKLML...")
+
+set(MKLML_RELEASE v0.17-rc)
```

Comment:
shouldn't this be the mkldnn release version?

Reply:
Yes. v0.17 is the release version of MKL-DNN. MKLML version is 2019.0.1 in this case, which is declared in `mklml/include/mkl_version.h`. Different releases of MKL-DNN may have the same package of MKLML (but with different download URL).